src/main/java/uk/gov/hmcts/sptribs/flag/refdata/FlagTypeService.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.sptribs.flag.refdata;
+
+import feign.FeignException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.sptribs.common.service.AuthorisationService;
+
+@Service
+@Slf4j
+public class FlagTypeService {
+
+    @Autowired
+    AuthorisationService authorisationService;
+
+    @Autowired
+    private FlagTypeClient flagTypeClient;
+
+    public Object getFlagTypes() {
+
```

Comment:
Same query as above, does it have to be Object? Can we not return some sort of Collection?

Reply:
I was just trying to understand response type, not used, I can remove